projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/src/commandsAndMenu.tsx

```diff
@@ -75,4 +76,22 @@ export function updateVDKMenu(commands: CommandRegistry) {
             }).catch((e) => console.log(e));
         },
     });
+
+    commands.addCommand("jp-vdk:menu-download", {
+        label: "Download",
+        caption: 'Execute VDK Download Command',
+        execute: async () => {
+            showDialog({
+                title: "Download Job",
```

Comment:
nit: maybe display the team set (in the background) for the user informatively - the unique identifier of a data job workload is team + name